getting-started/templates/systemlink-values.yaml

```diff
@@ -403,7 +404,7 @@ dashboardhost:
       plugins:
         ## Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loaded even if they lack a valid signature.
         ##
-        allow_loading_unsigned_plugins: "ni-slnotebook-datasource"
+        allow_loading_unsigned_plugins: ni-slnotebook-datasource,ni-sldataframe-datasource
```

Comment:
In this morning's meeting, we also included `ni-plotly-panel`. Is that something we need to include here @mure?

Looking at it more, all of these settings are in the default values.yaml file. Should these even be in here? In particular, I'm worried about the hard-coded version numbers for the plugins.

Reply:
Ah, I wasn't sure if that was one we only wanted internally.

Reply:
We should definitely add the plotly panel here too

Reply:
@spanglerco - I updated with the changes we discussed offline. Leaving this issue as unresolved until you sign-off.